packages/fxa-settings/src/lib/hooks/useNavigateWithQuery/index.tsx

```diff
@@ -9,6 +9,8 @@ export function useNavigateWithQuery() {
 
   return (to: string, options?: NavigateOptions<{}>) => {
     const location = window.location;
+    console.log('location', location);
+    debugger;
```

Comment:
@vpomerleau imma presume you did not mean to leave this here

Reply:
Indeed I didn't. Surprised we don't have a lint rule for this.